test/modules/amqp.py

```diff
@@ -40,6 +44,14 @@ def on_message(self, body, message):
     def on_conn_retry(self):
         LOG.error('Retrying connection for {0}'.format(self.__amqp_url))
 
+    def on_connection_error(self, exc, interval):
```

Comment:
@zyoung51 The connection retry should be reset by ensure_connection when the connection is reestablished (ensure_connection maintains it's own retry count and interval.)   The __max_error is a global threshold for the number of ensure_connection attempts.